op-batcher/batcher/driver.go

```diff
@@ -324,6 +331,66 @@ func (l *BatchSubmitter) loop() {
 	}
 }
 
+// checkRecentTxsOnStart Check to see if there was a batcher tx sent recently that
+// still needs more block confirmations before being considered finalized
+func (l *BatchSubmitter) checkRecentTxsOnStart() {
+	currBlock, err := l.Txmgr.BlockNumber(l.shutdownCtx)
+	if err != nil {
+		l.Log.Error("failed to retrieve current block number", "err", err)
+		return
+	}
+
+	currentBlock := new(big.Int).SetUint64(currBlock)
+	currentNonce, err := l.L1Client.NonceAt(l.shutdownCtx, l.Txmgr.From(), currentBlock)
+	if err != nil {
+		l.Log.Error("Failed to retrieve current nonce", "err", err)
+		return
+	}
+
+	rollupClient, err := l.EndpointProvider.RollupClient(l.shutdownCtx)
+	if err != nil {
+		l.Log.Error("Failed to get rollup client", "err", err)
+		return
+	}
+	rollupConfig, err := rollupClient.RollupConfig(l.shutdownCtx)
+	if err != nil {
+		l.Log.Error("Failed to retrieve rollup config", "err", err)
+		return
+	}
+	blockConfirms := rollupConfig.VerifierConfDepth
+
+	previousBlockBig := new(big.Int)
+	previousBlockBig.Sub(currentBlock, big.NewInt(int64(blockConfirms)))
```

Comment:
```suggestion
	previousBlockBig := new(big.Int).Sub(currentBlock, big.NewInt(int64(blockConfirms)))
```
maybe we also call it something like `oldestBlock` because it's not the _previous_ block.